definitions/checks/foreman/check_external_db_evr_permissions.rb

```diff
@@ -0,0 +1,56 @@
+module Checks
+  module Foreman
+    class CheckExternalDbEvrPermissions < ForemanMaintain::Check
+      metadata do
+        label :external_db_evr_permissions
+        for_feature :foreman_database
+        description 'Check that external DBs have proper EVR extension permissions'
+        tags :pre_upgrade
+        confine do
+          feature(:foreman_database) && !feature(:foreman_database).local? && feature(:katello)
+        end
+      end
+
+      def run
+        return unless evr_exists?
```

Comment:
Should we explicitly return true since the check would pass? I am unsure of how it handles `nil` returns from checks but `nil` is often a negative case.

Reply:
In this case foreman maintain is accepting nil as a pass, but I'll return true.